src/qmozview_p.cpp

```diff
@@ -860,7 +860,7 @@ void QMozViewPrivate::createView()
         mView->SetListener(this);
         setScreenProperties(QGuiApplication::primaryScreen()->depth(),
                             QGuiApplication::primaryScreen()->physicalDotsPerInch(),
-                            QGuiApplication::primaryScreen()->logicalDotsPerInch());
+                            QGuiApplication::primaryScreen()->physicalDotsPerInch());
```

Comment:
The `mDensity` and `mDpi` members will now always contain the same value. Since they've both private, I'd suggest getting rid of `mDensity` entirely from `QMozViewPrivate`.

It could also be dropped as a parameter, but I can see there's also a sensible symmetry in keeping the same parameters as for `EmbedLiteView::setScreenProperties()`.

Reply:
Fixed.